src/components/Modal/components/Dialog/tests/Dialog.test.tsx

```diff
@@ -33,4 +33,13 @@ describe('<Dialog>', () => {
     trigger(dialog.find('FadeUp'), 'onEntered');
     expect(dialog.find('FadeUp').prop('onEntered')).toHaveBeenCalledTimes(1);
   });
+
+  it('sets the aria-modal attribute to true', () => {
+    const trapFocus = mountWithAppProvider(
+      <Dialog labelledBy="test" onClose={jest.fn()} in>
+        something
+      </Dialog>,
+    ).find('div.Modal');
+    expect(trapFocus.prop('aria-modal')).toBe(true);
```

Comment:
I'd probably do something like this here: 
```
).find('[role="dialog"]');
    expect(dialog.prop('aria-modal')).toBe(true);
```
Generally we try to avoid using specific markup for tests in case it changes but it's pretty safe that the role will stay.